packages/twentytwenty-theme/src/components/archive/archive-pagination.js

```diff
@@ -41,13 +41,10 @@ const paginate = (totalPages, currentPage) => {
 };
 
 const Pagination = ({ state, actions, libraries }) => {
-  const { totalPages } = state.source.get(state.router.link);
-  const { path, page, query } = libraries.source.parse(state.router.link);
+  const { path, query, totalPages, next, previous, page } = state.source.get(
```

Comment:
Shouldn't this be `route` instead of `path`?

Reply:
I thought that no? It's the `path` attribute of [`Data`](https://github.com/frontity/frontity/pull/253/files#diff-879b131cdb78f509b45efc097fac4ab7R68)

 If I understood it correctly, it was not supposed to change?

Reply:
oh, nevermind, you did mean to change the `path` to `route` there as well. All clear.